### DISCUSSION TITLE
Feature request: support transferring Core assets

### DISCUSSION THREAD

**Discussion by josip-fundl** [phantom/docs#230] [General]
Transferring Core NFT assets is still not enabled in Phantom. It would do me a huge favor if I was actually able to move the assets around.

**Comment by bfriel** [comment]
This is on the roadmap. It was originally deprioritized relative to Token Extensions and its incompatibility with Expo (which Phantom is based on). Will follow up and see if we can provide you an ETA @d-reader-josip

**Reply by josip-fundl** [reply]
ETA would be appreciated. Not necessarily because this should be done ASAP, but more so to ensure that ticket has been put into the pipeline.

Thank you! 🙏

**Reply by bfriel** [reply]
@d-reader-josip wanted to follow up and let you know that transfer and burn support is scheduled to ship by end of June.

**Reply by bfriel** [reply]
Hey @d-reader-josip , support for Core transfers and burns is merged and should go out in our next release. Still targeting end of June for full rollout.

**Reply by josip-fundl** [reply]
That's wonderful 🙂

**Reply by bfriel** [reply]
Transfer support is now live! https://x.com/phantom/status/1809282603767505232?s=46&t=r80M6O27f4uq2et5ER7tGQ
